.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -33,23 +33,23 @@ body:
     id: code-of-conduct
     attributes:
       label: Code of Conduct
-      description: By submitting this issue, you agree to follow our [Code of Conduct.](https://insightsengineering.github.io/teal.modules.general/CODE_OF_CONDUCT.html)
+      description: By submitting this issue, you agree to follow our [Code of Conduct.](https://insightsengineering.github.io/teal.modules.general/latest-tag/CODE_OF_CONDUCT.html)
       options:
         - label: I agree to follow this project's Code of Conduct.
           required: true
   - type: checkboxes
     id: contributor-guidelines
     attributes:
       label: Contribution Guidelines
-      description: By submitting this issue, you agree to follow our [Contribution Guidelines.](https://insightsengineering.github.io/teal.modules.general/CONTRIBUTING.html)
+      description: By submitting this issue, you agree to follow our [Contribution Guidelines.](https://insightsengineering.github.io/teal.modules.general/latest-tag/CONTRIBUTING.html)
       options:
         - label: I agree to follow this project's Contribution Guidelines.
           required: true
   - type: checkboxes
     id: security-policy
     attributes:
       label: Security Policy
-      description: By submitting this issue, you agree to follow our [Security Policy.](https://github.com/insightsengineering/teal.modules.general/security/policy)
+      description: By submitting this issue, you agree to follow our [Security Policy.](https://github.com/insightsengineering/teal.modules.general/latest-tag/security/policy)
```

Comment:
I don't think we need to add latest-tag in templates . and this link does not exists

Reply:
Yes, sorry